### ISSUE TITLE
xx30-legacy boards coreboot's cbfs_size

### ISSUE THREAD

**Issue by tlaurion** [osresearch/heads#1338]
Just dumped backup of Lenovo BIOS and extracted the ifd layout following latest tests of #1292 to make sure no regression on x230-legacy flash boards (again, thanks for investigation from @rbreslow for T440p which led to understand that IFD BIOS region SHOULD match CBFS_SIZE under coreboot unless something else requires to reduce it).

No problem flashing x230-legacy-flash externally on top SPI flash: booted. 
New shared generic custom boot script shows, but flashing locally build ROM in defined IFD BIOS region of 7mb resulted in x230-hotp-legacy ROM flashed not booting the x230.

--- 

So here are some notes on extracted IFD information from bottom SPI backup:
```

user@heads-tests:~/heads$ ~/heads/build/x86/coreboot-4.13/util/ifdtool/ifdtool --layout /tmp/layout.txt ~/QubesIncoming/Insurgo/bottom_backup.rom 
File /home/user/QubesIncoming/Insurgo/bottom_backup.rom is 8388608 bytes
Wrote layout to /tmp/layout.txt
user@heads-tests:~/heads$ cat /tmp/layout.txt
00000000:00000fff fd
00500000:00bfffff bios
00003000:004fffff me
00001000:00002fff gbe
```

Then 
00bfffff - 00500000 = 6FFFFF (should be in CBFS_SIZE)
This in decimal is 7340031

Will retest if again a brick from #1292 dowloaded ROMs

**Comment by tlaurion** [comment]
This was flashed successfully merged on top of #1292.
Retesting #1292 alone

**Comment by tlaurion** [comment]
70000 should be good. Retested #1292 and success there.
